src/utils/XmlrpcClient.js

```diff
@@ -22,10 +22,11 @@ class XmlrpcCall {
           reject(err);
         }
         else if (resp[0] !== 1) {
-          const error = new Error('ROS XMLRPC Error');
+          const msg = resp[1];
+          const error = new Error(`ROS XMLRPC Error: ${msg}`);
```

Comment:
Is this fixing a separate bug?

Reply:
This isn't a bug - this is a response from a remote XMLRPC server that indicates some problem in handling the request. It will happen if you try to get a parameter or lookup a service that doesn't exist.

http://wiki.ros.org/ROS/Master_Slave_APIs